actionmailer/lib/action_mailer/test_helper.rb

```diff
@@ -123,9 +123,15 @@ def assert_enqueued_emails(number, &block)
     #       ContactMailer.with(email: 'user@example.com').welcome.deliver_later
     #     end
     #   end
-    def assert_enqueued_email_with(mailer, method, args: nil, queue: ActionMailer::Base.deliver_later_queue_name || "default", &block)
+    def assert_enqueued_email_with(mailer, method, params: nil, args: nil, queue: ActionMailer::Base.deliver_later_queue_name || "default", &block)
+      if mailer.is_a? ActionMailer::Parameterized::Mailer
+        params = mailer.instance_variable_get(:@params)
```

Comment:
I can't figure out why these instance variables aren't accessible. They [appear to be configured as attr_accessor](https://github.com/higher-pixels/rails/blob/cefabdff0783350193f969cec996010b52ddedb9/actionmailer/lib/action_mailer/parameterized.rb#L91)?